sdr-replication.gemspec

```diff
@@ -16,9 +16,8 @@ Gem::Specification.new do |s|
   s.required_rubygems_version = '>= 2.2.1'
 
   # Runtime dependencies
-  s.add_dependency 'json_pure', '~> 1.8'
   s.add_dependency 'rest-client', '~> 1.6'
-  s.add_dependency 'moab-versioning', '~> 1.3'
+  s.add_dependency 'moab-versioning', '~> 2.0'
```

Comment:
I don't know anything about 2.0, the last release that I'm aware of is 1.4 (I think).  So I can't evaluate this without dedicated time to evaluate the 2.0 release.